s3-test/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>s3-test</artifactId>
+
+    <parent>
+        <groupId>com.hazelcast.jet.tests</groupId>
+        <artifactId>hazelcast-jet-ansible-tests</artifactId>
+        <version>3.2-SNAPSHOT</version>
+    </parent>
+
+
+    <properties>
+        <main.basedir>${project.parent.basedir}</main.basedir>
+        <hadoop.version>2.7.4</hadoop.version>
```

Comment:
Copy-paste issue. We do not need `hadoop.version` here. However we should set version of `software.amazon.awssdk` as property.